### ISSUE TITLE
fixed new shorts category on home page

### ISSUE THREAD

**Issue by CheemsGalaxy** [gijsdev/ublock-hide-yt-shorts#37]
hi everyone!
If you're having trouble with YouTube's new Shorts layout, don't worry. I'll help you!
copy this code: https://anotepad.com/notes/iiaqypgg
I just added 2 lines of code from the author's original code to fix them.
YouTube is changing too fast; this code may no longer be effective.
![beforehide](https://github.com/user-attachments/assets/2efc1ded-4068-46df-860e-bc8afae746ad)

**Comment by gijsdev** [comment]
The Shorts button in the sidebar is also showing, so it looks like something is wrong on your end. Or did you intentionally remove that line from the list?

What browser and extension are you using? And what resolution is this on? I tried looking for this section but I can't find it. Could also be something that YouTube is slowly rolling out, of course.

**Comment by CheemsGalaxy** [comment]
> hi everyone! If you're having trouble with YouTube's new Shorts layout, don't worry. I'll help you! copy this code: https://anotepad.com/notes/iiaqypgg I just added 2 lines of code from the author's original code to fix them. ![beforehide](https://github.com/user-attachments/assets/2efc1ded-4068-46df-860e-bc8afae746ad)

**Comment by CheemsGalaxy** [comment]
edededddddddddddddedêde

**Comment by CheemsGalaxy** [comment]
> The Shorts button in the sidebar is also showing, so it looks like something is wrong on your end. Or did you intentionally remove that line from the list?
> 
> What browser and extension are you using? And what resolution is this on? I tried looking for this section but I can't find it. Could also be something that YouTube is slowly rolling out, of course.

yes, I intentionally removed that line and made improved the code
